### ISSUE TITLE
User is automatically joined to a group when writing a review

### ISSUE THREAD

**Issue by brentvdw** [boonebgorges/bp-group-reviews#11]
First, awesome plugin! Perfect for what we're doing. 

Got one issue for you to consider. Everytime a user writes a review on a group, it automatically makes that user join that group. Would you consider removing this from happening? As not all users want to join groups they are reviewing. 

Cheers
Brent

**Comment by boonebgorges** [comment]
Good idea, Brent. The auto-joining is a BP-core thing (any time you post content in a public group, you are auto-joined). It might be worth posting a ticket on trac.buddypress.org to ask about a toggle for this.
